boswatch/inputSource/lineInInput.py

```diff
@@ -40,21 +40,7 @@ def _runThread(self, dataQueue, lineInConfig, decoderConfig):
             lineInProc.setStderr(open(paths.LOG_PATH + "asla.log", "a"))
             lineInProc.start()
 
-            mmProc = ProcessManager(str(lineInConfig.get("mmPath", default="multimon-ng")), textMode=True)
-            if decoderConfig.get("fms", default=0):
-                mmProc.addArgument("-a FMSFSK")
-            if decoderConfig.get("zvei", default=0):
-                mmProc.addArgument("-a ZVEI1")
-            if decoderConfig.get("poc512", default=0):
-                mmProc.addArgument("-a POCSAG512")
-            if decoderConfig.get("poc1200", default=0):
-                mmProc.addArgument("-a POCSAG1200")
-            if decoderConfig.get("poc2400", default=0):
-                mmProc.addArgument("-a POCSAG2400")
-            if lineInConfig.get("mmChar"):
-                mmProc.addArgument("-C " + str(lineInConfig.get("mmChar")))
-            mmProc.addArgument("-f alpha")
-            mmProc.addArgument("-t raw -")
+            mmProc = self.startmm(decoderConfig)
             mmProc.setStdin(lineInProc.stdout)
```

Comment:
Wenn man der `startmm` direkt gleich den stdIn in den Funktionsaufruf mit übergibt, könnte man die anderen Befehle auch noch wegoptimieren. Dann kann die Funktion ``startmm` direkt eine laufende Instanz zurückliefern, ohne dass man die noch explizit starten muss.